### PR TITLE
Auto-add Note: prefix

### PR DESCRIPTION
**Pull request by wbamberg**
This is part of https://github.com/mdn/content/issues/8180.

At this point almost all the remaining unconvertible notes are just missing the `<strong>Note:</strong>` prefix. This auto-adds it.

In some cases this would not make sense, for example if the note is like `<p>Note that ...</p>`, we don't want: `<p><strong>Note:</strong> Note that ...</p>`. So I've excluded these from this PR and will take care of them in a follow-up, along with any other stragglers.

### CODE REVIEW

File: files/en-us/web/api/keyboardevent/keyboardevent/index.html [hunk 1]
```diff
@@ -79,7 +79,7 @@ <h3 id="Values">Values</h3>
 		</ul>
 
 		<div class="note">
-			<p><em>The <code>KeyboardEventInit</code> dictionary also accepts fields from
+			<p><strong>Note:</strong> <em>The <code>KeyboardEventInit</code> dictionary also accepts fields from
```
Comment (teoli2003) [inline files/en-us/web/api/keyboardevent/keyboardevent/index.html:82]: The `<em>` / `</em>` doesn't match our style guide. Can you fix it?

File: files/en-us/web/api/mouseevent/mouseevent/index.html [hunk 2]
```diff
@@ -141,7 +141,7 @@ <h3 id="Values">Values</h3>
       client fields will throw a <code>TypeError</code>.</p>
 
     <div class="note">
-      <p><em>The <code>MouseEventInit</code> dictionary also accepts fields from
+      <p><strong>Note:</strong> <em>The <code>MouseEventInit</code> dictionary also accepts fields from
```
Comment (teoli2003) [inline files/en-us/web/api/mouseevent/mouseevent/index.html:144]: Same here `<em>`s to remove

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Two cases of `<em>`/`</em>` enclosing the full note to remove, else evrerything is fine.

**Comment by wbamberg** [comment]
Thanks! Those should be fixed.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**